docs/02-sygma-sdk/04-Advanced/01-local-setup.md

```diff
@@ -4,7 +4,88 @@ id: advanced-localsetup
 title: Setting Up Locally
 description: The following section details how to setup Sygma locally.
 sidebar_position: 1
-draft: true
+draft: false
 ---
 
-More to come soon...
\ No newline at end of file
+### Local Sygma setup
+
+This setup contains preconfigured resources including: two EVM networks with Sygma contracts, one Substrate network with Sygma pallets, three relayers and a fee oracle. More detail can be found in local configuration.
```

Comment:
Small updates to blurb: 

"This setup contains preconfigured resources including: two EVM networks with Sygma contracts already deployed, one Substrate network with Sygma pallets implemented, three relayers listening for and executing on events, and a fee oracle. More details can be found in the local configuration."